app/src/main/java/org/simple/clinic/enterotp/EnterOtpUpdate.kt

```diff
@@ -31,7 +32,7 @@ class EnterOtpUpdate(
   private fun effectsForStateChanged(stateChanged: ProtectedState): Next<EnterOtpModel, EnterOtpEffect> {
     return when (stateChanged) {
       is Allowed -> dispatch(generateEffectForAllowingOtpEntry(stateChanged), AllowOtpEntry)
-      else -> // do something
+      is Blocked -> dispatch(BlockOtpEntryUntil(stateChanged.blockedTill))
```

Comment:
Same comment as before, where we can have just one UI effect for changing the OTP entry mode.

Reply:
Ah makes sense, updating this.